docs/running.rst

```diff
@@ -244,6 +244,34 @@ for an example of how to use this.
 
   There is only supported on Solana.
 
+Interacting with Solana running programs on-chain
+_________________________________________________
+
+  solang solana [SUBCOMMAND] [OPTIONS]... <PROGRAM_LOCATION>
+
+This means that the command line is ``solang solana`` followed by a subcommand 
+followed by any of the options described below and then the program location.
+
+General Options (for all subcommands):
+
+\-\-output-json
+  Specifies whether to export the call output in JSON format.
+
+\-\-help, -h
+  This displays a short description of all the options.
+
+Subcommands:
+
+  solang solana deploy [OPTIONS]... <PROGRAM_LOCATION>
+
+Deploys a program to Solana.
+
+Options specific to the ``deploy`` subcommand:
+
+\-\-verbose, -v
+  Specifies whether to display verbose program deployment information
+  (Conflicts with ``--output-json``).
+
```

Comment:
`deploy` is a subcommand, but the way you wrote it makes it look like it is another command. Instead of starting the section with the command, write first about what it does and its options, then give an usage example.

Reply:
good point